### PR TITLE
## Motivation and Context

### PR DESCRIPTION
**Pull request by dhruvbatra**
Cleaning up README

## How Has This Been Tested

Visually. 

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by erikwijmans — APPROVED** [review]

**Review by Skylion007 — APPROVED** [review]
Approved once pre-commit is done. We really should look into pre-commit.ci that will add these fixes to PRs automatically. :)

**Comment by Skylion007** [comment]
@dhruvbatra Please "squash" your commits when merging by clicking the dropdown next to the merge button.

**Comment by dhruvbatra** [comment]
My bad.

**Pull request merged**
